health-services/project-factory/src/server/processFlowClasses/user-processClass.ts

```
@@ -0,0 +1,225 @@
+        const url = config.host.hrmsHost + config.paths.hrmsEmployeeCreate;
+        const RequestInfo : any = defaultRequestInfo?.RequestInfo;
+        RequestInfo.userInfo.uuid = userUuid;
+        const requestBody = {
+            RequestInfo,
+            Employees: users,
+        };
+
+        try {
+            const response = await httpRequest(url, requestBody);
+            const map: any = {};
+            for (const user of response?.Employees) {
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)